### ISSUE TITLE
Is there a working INT implementation?

### ISSUE THREAD

**Issue by agrml** [p4lang/p4-applications#39]
I'm trying to launch INT application. After fixing a few typos in Python code I can launch INT. It sends packets, but monitor threads launch but does not get any traffic. So GUI shows nothing.

Can you point me on a working commit? Or it would be awesome if you have a working VM image with INT. I need to do a few measurements for my diploma project.

**Comment by zenhox** [comment]
I get the same problem too.  I try my best to install INT demo in p4 factory.After fixed some bugs, the html pages do not show any traffic.
What's wrong?     
It seems that the demo doesn't run correctly now!

**Comment by agrml** [comment]
You're right. I've also fixed bugs that were breaking interpretation, but it didn't help and the webpage was empty. Sad to say, but I didn't do anything wirh the code after this conclusion.

**Comment by euclidesfjr** [comment]
Hi, I'm learning the p4lang language, I need to do an implementation with INT (In-band Network Telemetry), containing 4 switches (2 spines and 2 leafs) and 4 hosts. Can anyone help me how can I do it? Ous if you have any repository that contains a similar configuration?

Thaks alot,

**Comment by jafingerhut** [comment]
@jklr Sorry to bother you with a ping on this old issue, but it seemed better to ask here than create a new issue with exactly the same topic.

Is there any published list of INT implementations that you know of?  Or in particular, is there any public P4_16 INT implementation that works on the open source v1model architecture, even if it is partial?

**Comment by jklr** [comment]
@jafingerhut 
https://github.com/opennetworkinglab/onos/blob/master/pipelines/basic/src/main/resources/int.p4
This implementation in the ONF repo might be the closest. It implements INT v1.0 using p4_16. I think it works with the BMv2 target too. @JonghwanHyun is the author. He may share more details.
